manifest.html

```diff
@@ -350,6 +352,48 @@ <h5>Looking for webinars...</h5></div>
         </div>
     </div>
 </section>
+
+<!--FAQ-->
+<section id="FAQ" class="section-header">
+  <div class="container">
+    <div class="row row-centered">
+        <div class="col-lg-12 headerContainer col-centered">
+            <span class="bottom-border bottom-border-blue center"><h1 class="section-header text-center ">Frequently Asked Questions</h1></span>
+        </div>
+    </div>
+
+    <ul class="collapsible" data-collapsible="faq-data">
+
+    <li>
+      <div class="collapsible-header faqTxt">QUESTION<i class="material-icons" id="ci">expand_more</i></div>
+      <div class="collapsible-body faqTxt"><span>ANSWER</span></div>
+    </li>
+    <li>
+      <div class="collapsible-header faqTxt">QUESTION<i class="material-icons" id="ci">expand_more</i></div>
+      <div class="collapsible-body faqTxt"><span>ANSWER</span></div>
+    </li>
+    <li>
+      <div class="collapsible-header faqTxt">QUESTION<i class="material-icons" id="ci">expand_more</i></div>
+      <div class="collapsible-body faqTxt"><span>ANSWER</span></div>
+    </li>
+    <li>
+      <div class="collapsible-header faqTxt">QUESTION<i class="material-icons" id="ci">expand_more</i></div>
+      <div class="collapsible-body faqTxt"><span>ANSWER</span></div>
+    </li>
+    <li>
+      <div class="collapsible-header faqTxt">QUESTION<i class="material-icons" id="ci">expand_more</i></div>
+      <div class="collapsible-body faqTxt"><span>ANSWER</span></div>
+    </li>
+    <li>
+      <div class="collapsible-header faqTxt">QUESTION<i class="material-icons" id="ci">expand_more</i></div>
+      <div class="collapsible-body faqTxt"><span>ANSWER</span></div>
+    </li>
```

Comment:
Too much Raw code here, this type content is usually handled by JavaScript. 
Read the implementation photos, events or webinars.